samples/sample_common/src/vaapi_allocator.cpp

```diff
@@ -332,7 +333,26 @@ mfxStatus vaapiFrameAllocator::AllocImpl(mfxFrameAllocRequest *request, mfxFrame
             {
                 format = VA_RT_FORMAT_RGBP;
             }
+#ifdef ENABLE_V4L2_SUPPORT
+            if ((m_export_mode & vaapiAllocatorParams::PRIME) && (format == VA_RT_FORMAT_YUV422))
```

Comment:
why VA_RT_FORMAT_YUV422 only?

Reply:
for V4L2 we used only YUY2/UYVY, so no need another formats

Reply:
This is an allocator. It's transparent to a fixed range FOURCC particular cameras can output.